.github/workflows/repro.yml

```diff
@@ -0,0 +1,63 @@
+---
+# https://docs.corelightning.org/docs/repro
+name: Repro Build Nightly
+on:
+  # 05:00 Berlin, 03:00 UTC, 23:00 New York, 20:00 Los Angeles
+  schedule:
+    - cron: "0 3 * * *"
+jobs:
+  ubuntu:
+    name: "Ubuntu repro build: ${{ matrix.version }}"
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false    # Let each build finish.
+      matrix:
+        version: ['focal', 'jammy', 'noble']
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
```

Comment:
Can we update it to [version 4](https://github.com/actions/checkout?tab=readme-ov-file#checkout-v4)? Current checkout v2 is throwing two warnings per ubuntu version: `The following actions use a deprecated Node.js version and will be forced to run on node20: actions/checkout@v2`.

Reply:
Updated to `actions/checkout@v4`.